gammapy/image/utils.py

```diff
@@ -799,7 +833,24 @@ def make_header(nxpix=100, nypix=100, binsz=0.1, xref=0, yref=0,
 
     Parameters
     ----------
-    TODO
+    nxpix : int
+	Number of pixels in x axis. Default is 100.
```

Comment:
Ok, I understand your concern, and I agree that it is easy to forget updating the doc string if the default is changed. I was emulating the behavior of the scipy covolve function:
 http://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.filters.convolve.html
Wht if we substitute the hard-coded value with the actual variable? Would that work with sphinx?